pkg/app/src/pages/organisations/index.tsx

```diff
@@ -73,21 +84,26 @@ export function OrganisationPage() {
 									variant="outlined"
 									onClick={handleClickCreate}
 								>
-									Create
+									{t('button:ui:create')}
 								</Button>
 							</Grid>
 						</Grid>
 						<FiltersSection
 							setFilters={setFilters}
-							setSortOption={setSortOption}
 							sortOptions={data?.displayValues?.sortOptions?.concat([])}
+							showFilters={enabledFeature?.ORGANIZATION_PAGE_SHOW_FILTERS}
+							showSearch={enabledFeature?.ORGANIZATION_PAGE_SHOW_SEARCH}
+							showSort={enabledFeature?.ORGANIZATION_PAGE_SHOW_SORT}
+							searchPlaceHolder={t('page:organisations:search_place_holder')}
+							ListTab={OrganizationFiltersListTab}
 						/>
 					</Box>
 					{paginatedData?.length === 0 && !loading && (
 						<Box sx={{ mt: 2, mb: 4 }}>
-							<Typography fontWeight={700}>No organisation found</Typography>
+							<Typography fontWeight={700}>{t('page:organisations:no_organisations')}</Typography>
 							<Typography>
-								No results found for “{filters}”. Try checking for typos or using a different term.
+								{t('page:organisations:no_result')} “{filters?.query}”.
```

Comment:
Please use Interpolation. https://www.i18next.com/translation-function/interpolation
Otherwise it is hard for the translators to understand the context